app/src/main/res/menu/menu_main.xml

```diff
@@ -13,6 +13,11 @@
         android:showAsAction="never"
         android:title="@string/export_messages"
         app:showAsAction="never" />
+    <item
+        android:id="@+id/show_archived"
+        android:showAsAction="never"
+        android:title="Show archived messages"
```

Comment:
You've missed moving it to strings.xml. And I think "messages" should be changed to "conversations" to keep consistency between strings.

Reply:
Good catch